mlflow/store/artifact/uc_volume_artifact_repo.py

```diff
@@ -107,17 +115,28 @@ def _download(self, output_path: str, file_path: str):
             file_path: The absolute path of the file to download.
 
         Returns:
-            The response from the API.
+            None
 
         See also:
             https://docs.databricks.com/api/workspace/files/download
         """
-        endpoint = f"{FILES_API_ENDPOINT}{file_path}"
-        with open(output_path, "wb") as f:
-            with self._api_request(endpoint=endpoint, method="GET", stream=True) as resp:
-                for content in resp.iter_content(chunk_size=DOWNLOAD_CHUNK_SIZE):
-                    f.write(content)
-                return resp
+        self.workspace_client = (
+            _get_databricks_sdk_workspace_client_if_experimental_files_api_enabled(
+                self.databricks_profile_uri
+            )
+        )
+        if self.workspace_client:
+            _logger.debug("Using Databricks SDK experimental Files API for file download.")
+            with open(output_path, "wb") as f:
+                with self.workspace_client.files.download(file_path) as remote_file:
+                    f.write(remote_file.contents.read())
```

Comment:
```suggestion
                with self.workspace_client.files.download(file_path) as remote_file:
                    for content in remote_file.contents:
                         f.write(content)
```

Can we write in chunks to avoid OOM?